app/src/views/GrantRegistryGrantDetail.vue

```diff
@@ -192,6 +191,8 @@ function useGrantDetail() {
       tx = await registry.updateGrant(g.id, owner, payee, metaPtr);
     }
 
+    // TODO: show waiting state screen
```

Comment:
Is the intent that we want this screenshot to be a component that takes a transaction hash and shows one of the 3 below states? I'm not clear on how we want to do this and asked about in Discord here: https://discord.com/channels/562828676480237578/838580829482647572/882309865119313930

![image](https://user-images.githubusercontent.com/17163988/131575894-a3ef8f7c-b919-4faa-8198-aa68125855f9.png)

Reply:
Yup tha's what I thought !